stripper/ze_ffxii_westersand_v8zeta1k.cfg

```diff
@@ -1,156 +1,1464 @@
-// fix chaos summon during epic & god mode
+;------------------------------------------------------------
+;STRIPPER CFG BY NOCTALI http://steamcommunity.com/id/h4sard
+;------------------------------------------------------------
+;-----------------------
+;OPEN SHIP VENT EARLIER
+;-----------------------
 modify:
 {
 	match:
 	{
-		"targetname" "Summon_Chaos_Temp"
-		"classname" "point_template"
+		"model" "*41"
+		"origin" "8656 -6464 6495.88"
+		"targetname" "Ship_Gate_End"
+		"classname" "func_movelinear"
 	}
 	delete:
 	{
-		"Template01" "Chaos_Summon_*"
+		"OnUser1" "Airship_End_Breakable1Break11"
 	}
 	insert:
 	{
-		"Template01" "Chaos_Summon_Model"
-		"Template02" "Chaos_Summon_Move"
-		"Template03" "Chaos_Summon_Particle_1"
-		"Template04" "Chaos_Summon_Physbox"
-		"Template05" "Chaos_Summon_Relay_Meteor"
-		"Template06" "Chaos_Summon_Rotate"
-		"Template07" "Chaos_Summon_Trigger_1"
-		"Template08" "Chaos_Summon_Wall"
+		"OnUser1" "Airship_End_Breakable1Break01"
 	}
 }
 
-// fix dragon boost
+;---------------
+;FIX BAD SHADOW
+;---------------
 modify:
 {
 	match:
 	{
-		"targetname" "Warmup_Fly_Push"
+		"targetname" "Ehe_Wall"
+		"classname" "prop_dynamic"
+	}
+	replace:
+	{
+		"disablereceiveshadows" "1"
+	}
+	insert:
+	{
+		"disableshadows" "1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"targetname" "Ehe_Wall"
+		"classname" "func_wall_toggle"
+	}
+	replace:
+	{
+		"disableshadows" "1"
+		"disablereceiveshadows" "1"
+	}
+}
+
+;-----------------
+;FIX INVISIBILITY
+;-----------------
+modify:
+{
+	match:
+	{
+		"origin" "-13568 13568 96"
+		"targetname" "Knife_Invis_Trigger"
 		"classname" "trigger_multiple"
 	}
 	delete:
 	{
-		"OnStartTouch" "!activatorAddOutputbasevelocity 0 -2100 15000-1"
+		"OnStartTouch" "!activatorAddOutputrendermode 60-1"
+		"OnStartTouch" "!activatorAddOutputrenderfx 100-1"
+		"OnStartTouch" "!activatorAddOutputrenderamt 00-1"
 	}
 	insert:
 	{
-		"OnStartTouch" "!activatorRunScriptCode{local vel=self.GetVelocity();self.SetVelocity(Vector(vel.x,vel.y-2100,vel.z+1500))}0-1"
+		"OnStartTouch" "!activatorAddOutputrendermode 100-1"
 	}
 }
```

Comment:
..is this known to even be broken? Verify you're not taking unneeded fixes from a completely different game.